microcosm_pubsub/dispatcher.py

```diff
@@ -30,40 +31,57 @@ def handle_batch(self, bound_handlers) -> List[MessageHandlingResult]:
 
         """
         return [
-            MessageHandlingResult.invoke(
-                func=self.handle_message,
-                message=message,
-                bound_handlers=bound_handlers
-            )
+            self.handle_or_ignore_message(message, bound_handlers)
```

Comment:
I am slowly moving the `MessageHandlingResult` inward so that we can capture more information (e.g. timing) for message handling. This necessarily involves moving the message context (e.g. `opaque`) outward.